### PR TITLE
routes for attend, sponsor, book in company

### PR DESCRIPTION
**Pull request by SarahPAms**

### CODE REVIEW

**Comment by arnoFleming** [comment]
see Slack #general for comment

**Comment by SarahPAms** [comment]
@arnoFleming it should be fixed

File: spec/features/attendee_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "attendees index action" do
+  let(:event) { create :event }
+
+  it "raises" do
```
Comment (arnoFleming) [inline spec/features/attendee_page_spec.rb]: name it better, why does it raise? `it "doesn't exist on our root path"` Or something like this?

File: spec/helpers/attendees_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (arnoFleming) [inline spec/helpers/attendees_helper_spec.rb]: please remove all files that do nothing useful.

File: spec/models/event_spec.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (arnoFleming) [inline spec/models/event_spec.rb]: please remove line

**Comment by arnoFleming** [comment]
@SarahPAms: you already have merge conflicts, please come and see me if you cant fix them yourself

**Pull request merged**